### PR TITLE
Updated README and ruby/npm dependencies

### PR DESCRIPTION
**Pull request by justin808**
Now, both the Rails and Webpack Dev servers start with
```
foreman start -f Procfile.dev
```

### CODE REVIEW

**Comment by justin808** [comment]
@mbreining Can you please review.

File: README.md [hunk 1]
```diff
@@ -1,40 +1,71 @@
...
+- If this work interests you and you are looking for full or part-time remote work, please
+  [click here](http://forum.railsonmaui.com/t/railsonmaui-is-hiring-and-partnering-part-time-remote-is-ok/156).
+
+A Full tutorial article can be found at: [Fast Rich Client Rails Development With Webpack and the ES6 Transpiler](http://www.railsonmaui.com/blog/2014/10/02/integrating-webpack-and-the-es6-transpiler-into-an-existing-rails-project/)
+Note, this source code repository is going to be ahead of the tutorial. We plan to update the tutorial soon.
 
 [Discussion forum regarding the tutorial](http://forum.railsonmaui.com/t/fast-rich-client-rails-development-with-webpack-and-the-es6-transpiler/82/10)
 
+# Example Application
+This is a simple example of using ReactJs for Comments that demonstrates the front end code in both
```
Comment (mbreining) [inline README.md]: How about?

"This is a simple example application that illustrates the use of ReactJs to implement a commenting system. Front-end code leverages both ReactJs and Rails asset pipeline while the backend is 100% Rails."

Or just rephrase that sentence, it wasn't clear for me...

File: README.md [hunk 2]
```diff
@@ -1,40 +1,71 @@
...
+Rails and using ReactJs, and has a Rails backend. It shows off a little bit of the interactivity
+of a ReactJs application, allowing the commment to choose the form layout. `react-bootstrap` is used
```
Comment (mbreining) [inline README.md]: [...] allowing the **commenter** to choose [...]?

**Comment by mbreining** [comment]
Looks good and works great. I tested the changes locally. Just a few minor comments re: README.

File: README.md (hunk 1 above)
Comment (justin808) [inline README.md]: good catch

**Pull request merged**